glean-core/src/upload/mod.rs

```diff
@@ -147,14 +147,20 @@ impl PingUploadManager {
     }
 
     /// Creates a `PingRequest` and adds it to the queue.
-    pub fn enqueue_ping(&self, document_id: &str, path: &str, body: JsonValue) {
+    pub fn enqueue_ping(
+        &self,
+        document_id: &str,
+        path: &str,
+        body: JsonValue,
+        debug_ping_value: Option<&String>,
```

Comment:
Nit: I think `debug_ping_tag` might be more descriptive for this variable.

Reply:
That's a mistake. During development I interchangeably used `debug_view_tag` and `debug_ping_tag`, but the latter was used by mistake. I thought I had removed all these cases, but clearly not :) Thansk for pointing that out!